src/tests/evas/meson.build

```diff
@@ -25,7 +25,7 @@ evas_suite_src = [
 
 evas_suite = executable('evas_suite',
   evas_suite_src,
-  dependencies: [evas_bin, evas, ecore_evas, dl, check, evas_ext_deps], #external deps needed here since tests do include internal headers
+  dependencies: [evas_bin, evas, ecore_evas, dl, check, evas_ext_none_static_deps, eet], #external deps needed here since tests do include internal headers
```

Comment:
This uses `dl` and `check` won't it be a problem?

Reply:
Not if they're empty `declare_dependency()` :D But thanks for pointing out, `check` here should be changed to `check_dep`.

Reply:
@JPTIZ are you gonna set `dl` as an empty `declare_dependency` somewhere?

Reply:
https://github.com/expertisesolutions/efl/pull/149#discussion_r442246746

Reply:
yeah, ahaha